spec/system/cart_billing_mode_spec.rb

```diff
@@ -8,71 +8,144 @@
   let!(:nonbillable_item) { create(:setup_item, facility:, billing_mode: "Nonbillable") }
   let!(:default_item) { create(:setup_item, facility:, billing_mode: "Default") }
   let!(:account) { create(:purchase_order_account, :with_account_owner, facility:) }
+  let!(:account_price_group_member) { create(:account_price_group_member, account: account, price_group: PriceGroup.globals.first) }
+  let!(:account_price_group_member_2) { create(:account_price_group_member, account: account, price_group: PriceGroup.globals.second) }
```

Comment:
If this works, I'd vote for using base and external for these:
```suggestion
  let!(:account_price_group_member) { create(:account_price_group_member, account: account, price_group: PriceGroup.base) }
  let!(:account_price_group_member_2) { create(:account_price_group_member, account: account, price_group: PriceGroup.external) }
```